pictopy/Lib/site-packages/pip/_internal/operations/freeze.py

```
@@ -0,0 +1,255 @@
+                    ):
+                        line = line.rstrip()
+                        if line not in emitted_options:
+                            emitted_options.add(line)
+                            yield line
+                        continue
+
+                    if line.startswith("-e") or line.startswith("--editable"):
+                        if line.startswith("-e"):
+                            line = line[2:].strip()
+                        else:
+                            line = line[len("--editable") :].strip().lstrip("=")
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)